skills/bear-security-setup/SKILL.md

```
@@ -0,0 +1,237 @@
+
+### 2. Configure psalm.xml
+
+Add taint plugin and stubs to existing `psalm.xml`.
+
+**Target configuration (based on user selection):**
+- `Page`: For `html` context serving web pages
+- `App`: For `api` context serving APIs
+
+```xml
+<?xml version="1.0"?>
+<psalm
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)